BookingsApi/BookingsApi.UnitTests/Mappings/V2/ParticipantToResponseV2MapperTests.cs

```diff
@@ -0,0 +1,61 @@
+using System.Collections.Generic;
+using BookingsApi.Contract.V1.Responses;
+using BookingsApi.Contract.V2.Responses;
+using BookingsApi.DAL.Helper;
+using BookingsApi.Domain;
+using BookingsApi.Domain.Enumerations;
+using BookingsApi.Domain.Participants;
+using BookingsApi.Domain.RefData;
+using BookingsApi.Mappings.V2;
+
+namespace BookingsApi.UnitTests.Mappings.V2
+{
+    public class ParticipantToResponseV2MapperTests
+    {
+        private readonly ParticipantToResponseV2Mapper _mapper = new ParticipantToResponseV2Mapper();
+
+        
+        [Test]
+        public void Should_map_participant_judge()
+        {
+            var caseRole = new CaseRole(5, "Judge");
+            var hearingRole = new HearingRole(13, "Judge") { UserRole = new UserRole(4, "Judge")};
+
+            var person = new PersonBuilder().Build();
+            var judge = new Judge(person, hearingRole, caseRole)
+            {
+                DisplayName = "Judge",
+                CreatedBy = "unit@hmcts.net"
+            };
+            judge.SetProtected(nameof(judge.CaseRole), caseRole);
+            judge.SetProtected(nameof(judge.HearingRole), hearingRole);
+
+            var response = _mapper.MapParticipantToResponse(judge);
+            
+            AssertParticipantCommonDetails(response, judge, caseRole, hearingRole);
+            response.Organisation.Should().BeNullOrWhiteSpace();
+        }
+
+        
+
+        private static void AssertParticipantCommonDetails(ParticipantResponseV2 response, Participant participant,
```

Comment:
please could you assert on these properties too, to cover all of them

HearingRoleCode
Organisation
LinkedParticipants

Reply:
I am not sure about the linked participant as I am testing a judge

Reply:
checked for null